weak_power/Mimic

```diff
@@ -1,3 +1,4 @@
+Mimic-3
```

Comment:
i think the 3 is supposed to be in brackets. other than that its good.

Reply:
I'll just do this since it's an oversight on my part